### PR TITLE
change-rules-improve

### PR DESCRIPTION
**Pull request by Joeper214**
### GOAL

- [x] add pagination for change rules modal
- [x] Add searching  for change rules modal

### CODE REVIEW

**Comment by Joeper214** [comment]
Please review and merge PR. @root-beer .. Thanks!. :bow:

File: app/views/infrastructures/modal/_modal_change_rules.html.erb [hunk 1]
```diff
@@ -30,8 +35,19 @@
             <td v-if="r.tags[0].key === 'Name'">{{r.tags[0].value}}</td>
             <td v-else></td>
           </tr>
+          <tr  v-if="filteredLength === 0 && dispItems.length > 0">
+            <td v-if="lang === 'en'" colspan="4"><h4> No results for <span class="text text-info">"{{filterKey}}"</span> was found. </h4></td>
+            <td v-if="lang === 'ja'"><h4> <span class="text text-info">"{{filterKey}}"</span> に一致する情報は見つかりませんでした。 </h4></td>
```
Comment (root-beer) [inline app/views/infrastructures/modal/_modal_change_rules.html.erb]: Could you make it colspan="4" in Japanese?

Comment (Joeper214) [inline app/views/infrastructures/modal/_modal_change_rules.html.erb]: :+1:   Yes. Sorry I forgot to consider.

**Comment by root-beer** [comment]
I tried to filter with 'zabbix'.
But results was divided into 2 pages.

sg1
sg2

**Comment by Joeper214** [comment]
:bow:  I'll fix this. thanks.

**Comment by Joeper214** [comment]
Does this PR still has any problems? @root-beer

**Comment by root-beer** [comment]
@Joeper214 When using filter, only the item of the displayed page is displayed though it exists in other pages.

**Comment by Joeper214** [comment]
:+1:  Thanks

**Comment by root-beer** [comment]
Does that make sense?

**Comment by Joeper214** [comment]
Yes. Please try my latest commit. Thanks. :bow:

**Comment by root-beer** [comment]
LGTM! Thanks. @Joeper214  :bow:

**Comment by Joeper214** [comment]
THANKS. :bow:

**Pull request merged**